config/rbac/extra-permissions-operator/namespaces.yaml

```diff
@@ -0,0 +1,15 @@
+- op: add
+  path: /rules/-
```

Comment:
where does this come from?

Reply:
Do you mean "why do the operator needs permission for namespaces"? If yes, because the `k8sattributes` needs it. And a SA cannot provide permissions over a resource it has not permissions.

Reply:
i mean was this manually generated?

Reply:
Im just not familiar with how this will be used

Reply:
> i mean was this manually generated?

Yes.

I was asked to apply the permissions for the operator in a similar way that users will apply them. https://github.com/open-telemetry/opentelemetry-operator/pull/2787/files/a12aade2316541c7f0be674704c53980378b2321#r1560704098